module/icp/algs/edonr/edonr.c

```diff
@@ -30,11 +30,7 @@
  */
 
 /* determine where we can get bcopy/bzero declarations */
-#ifdef	_KERNEL
-#include <sys/systm.h>
-#else
-#include <strings.h>
-#endif
+#include <sys/strings.h>
```

Comment:
looks like we aren't really "determining" anything anymore.

Reply:
Indeed.  I'll drop the comment when I'm here.